### PR TITLE
fix(chat): fix application name postfix

### PR DESCRIPTION
**Pull request by Derikyan**
**Description:**

Names of Applications should not contain postfix with version

Issues:

- Issue #2006

**UI changes**

Снимок экрана 2024-09-06 в 21 31 52

**Checklist:**

- [x] the pull request name complies with [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] the pull request name starts with `fix(<scope>):`, `feat(<scope>):`, `feature(<scope>):`, `chore(<scope>):`, `hotfix(<scope>):` or `e2e(<scope>):`. If contains breaking changes then the pull request name must start with `fix(<scope>)!:`, `feat(<scope>)!:`, `feature(<scope>)!:`, `chore(<scope>)!:`, `hotfix(<scope>)!:` or `e2e(<scope>)!:` where `<scope>` is name of affected project: `chat`, `chat-e2e`, `overlay`, `shared`, `sandbox-overlay`, etc.
- [x] the pull request name ends with `(Issue #<TICKET_ID>)` (comma-separated list of issues)
- [x] I confirm that do not share any confidential information like API keys or any other secrets and private URLs

### CODE REVIEW

**Comment by Derikyan** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2078.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by Derikyan** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2078.nightly-test.deltixhub.io
>E2E tests status: success

File: apps/chat/src/components/Common/ReplaceConfirmationModal/Components.tsx [hunk 1]
```diff
@@ -428,6 +428,7 @@ const ApplicationView = ({
     folderId: getFolderIdFromEntityId(application.name),
     type: EntityType.Application,
   };
+  const applicationName = application.name.split('__')[0];
```
Comment (Alexander-Kezik) [inline apps/chat/src/components/Common/ReplaceConfirmationModal/Components.tsx]: I think it's not correct. What if application was published through api? We need to make validation here that the last part is version and only in this case we should remove the last part. Also name could have __ and here it won't be shown

**Comment by Derikyan** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2078.nightly-test.deltixhub.io
>E2E tests status: success

File: apps/chat/src/utils/app/publications.ts [hunk 2]
```diff
@@ -45,6 +45,10 @@ export const createTargetUrl = (
     ...lastElement,
   );
 
+  if (featureType !== FeatureType.Chat && featureType !== FeatureType.Prompt) {
```
Comment (IlyaBondar) [inline apps/chat/src/utils/app/publications.ts]:
```suggestion
  if (featureType === FeatureType.Application) {
```

**Comment by Derikyan** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2078.nightly-test.deltixhub.io
>E2E tests status: success

**Review by IlyaBondar — APPROVED** [review]

**Pull request merged**